### PR TITLE
Fix tutorial

### PR DESCRIPTION
**Pull request by xiangmy**
- fix typo in part2/parameter
- fix errors in part2/memoryobject and part2/simplecache because getMatserPort() and getSlavePort() aren't used any mare

### CODE REVIEW

File: _static/scripts/part2/memoryobject/simple_memobj.hh [hunk 1]
```diff
@@ -42,6 +42,7 @@
  * This memobj is fully blocking (not non-blocking). Only a single request can
  * be outstanding at a time.
  */
+
```
Comment (powerjg) [inline _static/scripts/part2/memoryobject/simple_memobj.hh]: Any need for this change?

File: _static/scripts/part2/memoryobject/simple_memobj.hh [hunk 2]
```diff
@@ -90,6 +91,7 @@ class SimpleMemobj : public MemObject
         AddrRangeList getAddrRanges() const override;
 
         /**
+
```
Comment (powerjg) [inline _static/scripts/part2/memoryobject/simple_memobj.hh]: Need for this change?

File: _static/scripts/part2/simplecache/simple_cache.cc [hunk 3]
```diff
@@ -386,7 +374,7 @@ SimpleCache::insert(PacketPtr pkt)
 
         // Write back the data.
         // Create a new request-packet pair
-        RequestPtr req = new Request(block->first, blockSize, 0, 0);
+        RequestPtr req(new Request(block->first, blockSize, 0, 0));
```
Comment (powerjg) [inline _static/scripts/part2/simplecache/simple_cache.cc]: Why this change?

File: part2/simplecache.rst [hunk 4]
```diff
@@ -371,7 +372,7 @@ This data is guaranteed to be the size of the cache block since we made sure to
             auto block = std::next(cacheStore.begin(bucket),
                                    random_mt.random(0, bucket_size - 1));
 
-            RequestPtr req = new Request(block->first, blockSize, 0, 0);
+            RequestPtr req(new Request(block->first, blockSize, 0, 0));
```
Comment (powerjg) [inline part2/simplecache.rst]: Why this change?

**Comment by xiangmy** [comment]
Hi Jason,

Thanks for your advice. Sorry that I didn't make this a clear pull request. Do I need to pull request again to separate the commits and explain what I did?

File: _static/scripts/part2/memoryobject/simple_memobj.hh (hunk 1 above)
Comment (xiangmy) [reply _static/scripts/part2/memoryobject/simple_memobj.hh]: I guess I add this line by mistake. Sorry about that.

File: _static/scripts/part2/memoryobject/simple_memobj.hh (hunk 2 above)
Comment (xiangmy) [reply _static/scripts/part2/memoryobject/simple_memobj.hh]: I guess I add this line by mistake. Sorry about that.

File: _static/scripts/part2/simplecache/simple_cache.cc (hunk 3 above)
Comment (xiangmy) [reply _static/scripts/part2/simplecache/simple_cache.cc]: The original simple_cache.cc will generate the following error when I compile, so I change it.
```
[     CXX] X86/learning_gem5/simple_cache/simple_cache.cc -> .o
build/X86/learning_gem5/simple_cache/simple_cache.cc:378:14: error: no viable conversion from 'Request *' to 'RequestPtr' (aka 'shared_ptr<Request>')
RequestPtr req = new Request(block->first, blockSize, 0, 0);
```

File: part2/simplecache.rst (hunk 4 above)
Comment (xiangmy) [reply part2/simplecache.rst]: The same reason as described above

File: _static/scripts/part2/simplecache/simple_cache.cc (hunk 3 above)
Comment (powerjg) [reply _static/scripts/part2/simplecache/simple_cache.cc]: Ok.

**Comment by powerjg** [comment]
You don't need to do another PR. Just force push to your branch.
